data/common.yaml

```diff
@@ -18,9 +18,9 @@ profile_influxdb::config::logging_level: "warn"
 profile_influxdb::config::meta_dir: "/var/lib/influxdb/meta"
 profile_influxdb::config::monitor_store_interval: "60s"
 profile_influxdb::config::reporting_disabled: true
-profile_influxdb::config::ssl_ca_content: ""
-profile_influxdb::config::ssl_cert_content: ""
-profile_influxdb::config::ssl_key_content: ""
+profile_influxdb::config::ssl_ca_content: "need to set in Puppet"
+profile_influxdb::config::ssl_cert_content: "need to set in Puppet"
+profile_influxdb::config::ssl_key_content: "need to set in Puppet"
```

Comment:
Do we want to mention that we'd prefer that this value was encrypted with eyaml?

Reply:
this has been fixed